Userland/Applications/Presenter/PresenterWidget.cpp

```diff
@@ -83,21 +83,33 @@ ErrorOr<void> PresenterWidget::initialize_menubar()
             update_slides_actions();
         }
     });
-    TRY(presentation_menu->try_add_action(next_slide_action));
-    TRY(presentation_menu->try_add_action(previous_slide_action));
-    m_next_slide_action = next_slide_action;
-    m_previous_slide_action = previous_slide_action;
-
-    TRY(presentation_menu->try_add_action(GUI::Action::create("&Full Screen", { KeyModifier::Mod_Shift, KeyCode::Key_F5 }, { KeyCode::Key_F11 }, TRY(Gfx::Bitmap::try_load_from_file("/res/icons/16x16/fullscreen.png"sv)), [this](auto&) {
+    auto full_screen_action = GUI::Action::create("&Full Screen", { KeyModifier::Mod_Shift, KeyCode::Key_F5 }, { KeyCode::Key_F11 }, TRY(Gfx::Bitmap::try_load_from_file("/res/icons/16x16/fullscreen.png"sv)), [this](auto&) {
         this->window()->set_fullscreen(true);
-    })));
-    TRY(presentation_menu->try_add_action(GUI::Action::create("Present From First &Slide", { KeyCode::Key_F5 }, TRY(Gfx::Bitmap::try_load_from_file("/res/icons/16x16/play.png"sv)), [this](auto&) {
+    });
+    auto present_from_first_slide_action = GUI::Action::create("Present From First &Slide", { KeyCode::Key_F5 }, TRY(Gfx::Bitmap::try_load_from_file("/res/icons/16x16/play.png"sv)), [this](auto&) {
         if (m_current_presentation) {
             m_current_presentation->go_to_first_slide();
             update_web_view();
         }
         this->window()->set_fullscreen(true);
-    })));
+    });
+
+    if (!m_current_presentation) {
+        next_slide_action->set_enabled(false);
+        previous_slide_action->set_enabled(false);
+        full_screen_action->set_enabled(false);
+        present_from_first_slide_action->set_enabled(false);
+    }
+
+    TRY(presentation_menu->try_add_action(next_slide_action));
+    TRY(presentation_menu->try_add_action(previous_slide_action));
+    TRY(presentation_menu->try_add_action(full_screen_action));
+    TRY(presentation_menu->try_add_action(present_from_first_slide_action));
+
+    m_next_slide_action = next_slide_action;
+    m_previous_slide_action = previous_slide_action;
+    m_full_screen_action = full_screen_action;
+    m_present_from_first_slide_action = present_from_first_slide_action;
```

Comment:
What do you mean by that? Something like `m_next_slide_action = TRY(presentation_menu->try_add_action(next_slide_action));`? `try_add_action()` doesn't return the action, if that's the case

Otherwise, sorry for being such a n00b :)

Reply:
I think that Linus is suggesting to do things like:
```cpp
m_previous_slide_action = GUI::Action::create("&Previous", { KeyCode::Key_Left }, TRY(Gfx::Bitmap::try_load_from_file("/res/icons/16x16/go-back.png"sv)), [this](auto&) {
        if (m_current_presentation) {
            m_current_presentation->previous_frame();
            update_web_view();
            update_slides_actions();
        }
    });
```

Reply:
@LucasChollet thank you! When I first saw the message I though it might be this, but I decided it was better to ask anyway